### PR TITLE
Add Identity and Access Management into HCC settings procedure.

### PR DESCRIPTION
**Pull request by fmcdonal**

### CODE REVIEW

File: docs/kafka/access-mgmt-kafka/README.adoc [hunk 1]
```diff
@@ -543,7 +543,7 @@ endif::[]
 
 .Procedure
 
-. In the upper-right corner of the {product-kafka} web console, select the gear icon, and click *Settings* > *User Access* > *Groups*
+. In the upper-right corner of the {product-kafka} web console, select the gear icon, and click *Settings* > *Identity & Access Management* > *User Access* > *Groups*
```
Comment (smccarthy-ie) [inline docs/kafka/access-mgmt-kafka/README.adoc:546]:
```suggestion
. In the toolbar of the {product-kafka} web console, select the gear icon, and click *Settings* > *Identity & Access Management* > *User Access* > *Groups*.
```

File: docs/kafka/access-mgmt-kafka/README.adoc [hunk 2]
```diff
@@ -543,7 +543,7 @@ endif::[]
...
 . Click the name of the user group.
 . From the *Roles* tab, click *Add role* and select `User Access principal viewer`.
 . Click *Add to group* to add the role to the group.
```
Comment (smccarthy-ie) [inline docs/kafka/access-mgmt-kafka/README.adoc:549]:
```suggestion
. Click *Add to group*.
```

File: docs/registry/access-mgmt-registry/README.adoc [hunk 3]
```diff
@@ -211,7 +211,7 @@ endif::[]
 .Procedure
 
 . At the top of the {service-url-registry}[{registry} web console], select the gear icon.
```
Comment (smccarthy-ie) [inline docs/registry/access-mgmt-registry/README.adoc:213]:
```suggestion
. In the toolbar of the {service-url-registry}[{registry} web console], select the gear icon. 
```

File: docs/registry/access-mgmt-registry/README.adoc [hunk 4]
```diff
@@ -211,7 +211,7 @@ endif::[]
 .Procedure
 
 . At the top of the {service-url-registry}[{registry} web console], select the gear icon. 
-. Click *Settings*, then *User Access*, and then *Groups*.
+. Click *Settings*, then *Identity & Access Management*, then *User Access*, and then *Groups*.
```
Comment (smccarthy-ie) [inline docs/registry/access-mgmt-registry/README.adoc:214]:
```suggestion
. Click *Settings* > *Identity & Access Management* > *User Access* > *Groups*.
```

File: docs/kafka/access-mgmt-kafka/README.adoc (hunk 1 above)
Comment (fmcdonal) [reply docs/kafka/access-mgmt-kafka/README.adoc:546]: @smccarthy-ie I do not think we need to click "Settings" anymore?
Also, I think both guides should read the same first line and maybe Kafka should also say 
"At the top of the [Streams for Apache Kafka web console](https://console.redhat.com/application-services/streams/kafkas), select the gear icon."

File: docs/kafka/access-mgmt-kafka/README.adoc (hunk 2 above)
Comment (fmcdonal) [reply docs/kafka/access-mgmt-kafka/README.adoc:549]: @smccarthy-ie separating this into 2 steps like in registry.

File: docs/kafka/access-mgmt-kafka/README.adoc (hunk 1 above)
Comment (smccarthy-ie) [reply docs/kafka/access-mgmt-kafka/README.adoc]: @fmcdonal Good spot! Yes should be "click *Identity & Access Management* > *User Access* > *Groups*" :)

**Review by smccarthy-ie — APPROVED** [review]
LGTM, thanks!

File: docs/kafka/access-mgmt-kafka/README.adoc [hunk 5]
```diff
@@ -543,10 +543,11 @@ endif::[]
 
 .Procedure
 
-. In the upper-right corner of the {product-kafka} web console, select the gear icon, and click *Settings* > *User Access* > *Groups*
+. In the toolbar of the {service-url-kafka}[Openshift Streams for {product-kafka} web console], select the gear icon.
```
Comment (smccarthy-ie) [inline docs/kafka/access-mgmt-kafka/README.adoc:546]:
```suggestion
. In the toolbar of the {service-url-kafka}[{product-kafka} web console], select the gear icon.
```

**Review by smccarthy-ie — APPROVED** [review]
LGTM!

**Pull request merged**